perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="lidar_detection_mode" default="camera_lidar_fusion" description="options: `camera_lidar_fusion`, `lidar` or `camera`"/>
```

Comment:
I think you can remove this param since this file name is camera lidar radar fusion